fs/dafsserver/dafsserver.cpp

```diff
@@ -5463,6 +5474,8 @@ class CRemoteFileServer : implements IRemoteFileServer, public CInterface
 
         selecthandler->start();
 
+        unsigned timeLastLog = msTick();
```

Comment:
minor: if there is an exception within 1st min, this will mean it is not reported, it would probably be better to initialize to zero, forcing 1st call to trigger logging (and setting of timeLastLog)